CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# CHANGE LOG
+
+## VNext
```

Comment:
Use lower case first letter for `vNext`.

Reply:
Updated.